### ISSUE TITLE
[Question] I have to use unityLibrary without fuw.unitypackage.

### ISSUE THREAD

**Issue by Raonshi** [juicycleff/flutter-unity-view-widget#817]
Hello I'm developing flutter mobile application with unity library.

I have to use unityLibrary which exported without fuw.unitypackage because the unity library has made by other team(this team only export android and ios).

Are there any ways to integrate unityLibrary(exported by android or ios without fuw.unitypackage) and flutter_unity_widget?

**Comment by k0uhe1D** [comment]
Hello, while it is technically possible, I believe that it's best to use the fuw.unitypackage. 

The fuw.unitypackage includes the following:
- Custom build settings for calling the Unity library from Flutter ([FlutterUnityIntegration](https://github.com/juicycleff/flutter-unity-view-widget/tree/master/example/unity/DemoApp/Assets/FlutterUnityIntegration/Editor))

By manually applying these settings, it is possible to use the Unity library in a Flutter app even without the fuw.unitypackage. However, this requires advanced knowledge of native development. 

Therefore, while it is technically possible, I consider it to be challenging.

**Comment by Raonshi** [comment]
@k0uhe1D
Yeah. I agree too. actually, I already tried to export unityLibrary manually, but there were some gradle problem.

I just wanted easy way to integrate flutter and unityLibrary(exported without fuw.unityPackage).
But i recognized there is no easy way.

Anyway, Thanks for your answer. I'll try again🥲.
